### PR TITLE
Bugfix/lattice 2262 locks of doom

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt [hunk 1]
```diff
@@ -493,12 +485,10 @@ class PostgresEntityDataQueryService(
```
Comment (you-leee) [inline src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt]: `entityKeyIdsToLinkingIds` can be also deleted from here

File: src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt [hunk 2]
```diff
@@ -529,13 +518,11 @@ class PostgresEntityDataQueryService(
```
Comment (you-leee) [inline src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt]: `entityKeyIdsToLinkingIds` should be deleted here

File: src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt [hunk 3]
```diff
@@ -566,12 +552,10 @@ class PostgresEntityDataQueryService(
         val entityKeyIdsToLinkingIds = getLinkingIdsOfEntityKeyIds(replacementProperties.keys)
```
Comment (you-leee) [inline src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt]: `entityKeyIdsToLinkingIds` should be deleted here

**Comment by makosblade** [comment]
V.happy to not be passing around that connection anymore

**Comment by makosblade** [comment]
Oops lol, didn't mean to un-draft this 🤣

File: src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt (hunk 2 above)
Comment (geekbeast) [reply src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt]: I can't... quite yet. It is captured in by the function closure for tombstoneFn

Comment (you-leee) [reply src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt]: ah.. yeah, missed that, sorry!

**Pull request merged**